src/main/java/tahub/contacts/logic/commands/EditCommand.java

```diff
@@ -50,7 +50,6 @@ public class EditCommand extends Command {
 
     public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
     public static final String MESSAGE_NOT_EDITED = "At least one field to edit must be provided.";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
```

Comment:
Error message removed?

Reply:
Yep, it should not be possible to create a duplicate person since the person cannot edit matric number. See the commit body here for more details: https://github.com/AY2425S1-CS2103T-F14B-2/tp/pull/63/commits/ce34c53242648dd3d4421cce2033b7aa29c43446